src/styles/global.css

```
@@ -366,25 +367,45 @@ h5 {
 }
 
 .form-input:focus {
-  border: 3px solid var(--primary-color);
+  border: 2px solid var(--primary-color);
 }
 
 .form-input::placeholder {
   font-family: "Poppins";
   font-size: var(--font-size-text);
-  color: var(--text-color);
+  color: rbg(193, 193, 193);
```

Biome (2.1.2)

[error] Unexpected unknown function: rbg

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)